src/server.ls

```diff
@@ -13,7 +13,8 @@ module.exports =
 
   before-all: (done) ->
     mongo-db-name = "exosphere-mongo-service-#{env}"
-    MongoClient.connect "mongodb://localhost:27017/#{mongo-db-name}", N (mongo-db) ->
+    mongo-address = if env is \test then \localhost else \mongo
```

Comment:
Can we spin up the MongoDB Docker image for tests as well? That's the whole point of using Docker for the SDK. People shouldn't have to install all sorts of databases in order to work on an application.